### DISCUSSION TITLE
Possible to send installer code sections to panel?

### DISCUSSION THREAD

**Discussion by jthrilly** [ParadoxAlarmInterface/pai#522] [Q&A] [answered]
Hello all 👋 

I am looking into installing PAI with my SP6000+ panel at the moment. 

Since I will lose access to Babyware, Blueeye etc when using PAI, I am wondering if it is possible to send panel section codes (for example "955" to rescan modules) directly from PAI? The other use case I have would be for wireless sensor pairing.

I have a wireless keypad which makes programming painful, so I usually use the other software for this.

Thanks for your help.

**Comment by yozik04** [comment, marked as answer]
Hey @jthrilly,

Sending panel section codes is currently not supported and I doubt I will ever do that.

All this SP/MG will stay as is until we will find a maintainer with that kind of panel. I only have EVO192.

**Reply by jthrilly** [reply]
Thanks for the reply (and for all your work putting this together!). 

Can I ask why you doubt you would ever implement the feature? If the babyware protocol has been reverse engineered, couldn't common tasks (like the examples I gave above) be implemented the same way as the alarm functionality is? 

For what it is worth, I am potentially interested in helping out with the project. I am a React developer primarily, but I have a background in data science and am familiar with Python. Is there something I can do to help in the short term?

**Reply by yozik04** [reply]
I am spending minimal time to just keep it working. Current project state covers all my needs. SP/MG was maintained by @jpbarraca. I never touched that.

If you want, you can try to make sense of the protocol that Babyware sends. We have a decoder for that. Some info is in the wiki how to do that. If you come with a PR for your feature I will happily review that.

**Reply by jthrilly** [reply]
Thanks for your time. Once again, I really appreciate all the work you and the other collaborators put into this project. I will take a look at the code once I have my HA setup, and see if there's anything I can contribute.
